docs/GUIDE.md

```diff
@@ -0,0 +1,229 @@
+# A Warrior's Guide to Thunderdome
+
+Thunderdome is a fun way to perform agile scrum planning. See Wikipedia.
+
+As a new warrior in this realm, let this be your guide to Battles. First, we need to know who you are.
```

Comment:
We're more than just pointing battles ;)

Reply:
```suggestion
As a new warrior in this realm, let this be your guide. First, we need to know who you are.
```